src/index.js

```diff
@@ -445,7 +445,7 @@ class Datatable extends React.Component {
       body.push(
         <tr key={`${this.props.keyName}-row-zero-length`} className="tbody-tr-default">
           <td className="tbody-td-default" colSpan={this.props.tableHeader.length}>
-            No results to be shown.
+            {this.props.labels.no_results || 'No results to be shown.'}
```

Comment:
Would you like to change this to `this.props.labels.noResults`? Since I am using camelCase instead of underscores, I expect consistency across the entire library so people who use this won't be confused what API uses what kind of variable naming.

Reply:
Of course. I'll change it in a minute.

Reply:
Thank you!